lib/client.ts

```diff
@@ -67,23 +72,28 @@ function request(url: string, payload: any, method: string): Function {
 export class Client {
   private readonly host: string;
   private authorize: any;
+  private fetchConfig: RequestInit = {};
 
   constructor(authorizer: Authorizer, host: string = hostname) {
     this.authorize = authorizer.authorize
     this.host = host
```

Comment:
my only thought with that was that you would be forced to provide the host. which may not be a big deal. i've got mixed feelings